package/src/values/animation/spring/functions/spring.ts

```diff
@@ -6,34 +9,21 @@
  * @param velocity The initial velocity
  */
 export const createSpringEasing = (
-  params: Partial<{
-    mass: number;
-    stiffness: number;
-    damping: number;
-    velocity: number;
-  }>
-) => {
-  const {
-    mass,
-    stiffness,
-    damping,
-    velocity = 0,
-  } = {
+  params: Partial<SpringConfig>
+): TimingConfig => {
+  const config = {
     mass: 1,
     stiffness: 100,
     damping: 10,
+    velocity: 0,
     ...params,
   };
-  // TODO: Find correct velcoity
-  return getSpringEasing(mass, stiffness, damping, velocity / 100);
+  config.velocity /= 100;
```

Comment:
@chrfalch  what does this `/ 100` mean? Could it be because we provide the wrong velocity values?